templates/dart/test/query_test.dart.twig

```diff
@@ -1,14 +1,16 @@
+import 'dart:convert';
+
 import 'package:{{ language.params.packageName }}/{{ language.params.packageName }}.dart';
 {% if 'dart' in language.params.packageName %}
 import 'package:test/test.dart';
 {% else %}
 import 'package:flutter_test/flutter_test.dart';
 {% endif %}
 
-class BasicFilterQueryTest {
+class BasicFilterQueryTest<V> {
   final String description;
-  final dynamic value;
-  final String expectedValues;
+  final V value;
+  final List<V> expectedValues;
```

Comment:
This isn't always the case. If you pass a string, expected value is List of strings. If you pass in a list of strings, expected value is still list of strings.

Reply:
unresolved